### DISCUSSION TITLE
How to get last inserted row ID?

### DISCUSSION THREAD

**Discussion by LDMFD** [uptrace/bun#707] [Q&A] [answered]
Is LastInsertId supported - if not, how to get the last inserted row ID?

**Comment by vmihailenco** [comment, marked as answer]
The last inserted row ID should be automatically assigned on the model when it is inserted. `LastInsertId` is supported too if the driver provides it.

**Reply by LDMFD** [reply]
I just tested this and it's working. I have no idea why it didn't appear to be working for me previously. Apologies.

**Reply by LDMFD** [reply]
In fact for a column with spec `bun:"id,pk,autoincrement"` it seems that I don't even need to use `Returning("Id")` any more at all. Is it automatic in some cases?
